src/javascript/binary/pages/bet/tick_trade.js

```diff
@@ -153,7 +153,7 @@ var TickDisplay = function() {
                         if (data[i][0] === 'tick') {
                             var tick = {
                                 epoch: parseInt(data[i][1]),
-                                quote: parseFloat(data[i][2])
+                                quote: parseFloat(data[i][2]).toFixed(data[i][2].split('.')[1].length)
```

Comment:
i don't know if its valid in our case but if we don't send it as decimal value it will fail (just added a note, don't think we ever send it without decimal places)